python/cuml/cuml/experimental/accel/__main__.py

```diff
@@ -46,8 +48,16 @@
     default="converted_sklearn_model.pkl",
     help="Output path for the converted sklearn estimator file.",
 )
+@click.option("-v", "--verbose", count=True)
```

Comment:
I'd recommend adding a help message that maps the numbers of `v` to what `logger_level` is set

Reply:
Done in https://github.com/rapidsai/cuml/pull/6353/commits/d38f1e111a2f86f03d23d2fd45c161cb7f766152 .